src/stream.rs

```diff
@@ -254,6 +256,14 @@ impl StreamMap {
         self.peer_max_streams_uni = cmp::max(self.peer_max_streams_uni, v);
     }
 
+    /// Drops completed stream.
+    ///
+    /// This should only be called when Stream::is_complete() returns true for
+    /// the given stream.
+    pub fn collect(&mut self, _stream_id: u64, _local: bool) {
+        // TODO: do something here
```

Comment:
In the short term we'll just need to track completed streams count against the current limit, but in the future we can do additional clean-up (e.g. maybe free some memory or something like that).

Reply:
I made https://github.com/cloudflare/quiche/pull/164 to show how this is used (I decided to make two separate PRs to avoid a huge PR that is hard to review).